### PR TITLE
1124/show main categories

### PR DESCRIPTION
**Pull request by alireza-sharifpour**
- #1124

### CODE REVIEW

File: src/helpers/projects.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import { ICategory, IConvertedCategories } from '@/apollo/types/types';
```
Comment (RamRamez) [inline src/helpers/projects.ts:1]: since this file is not related to projects page, I think you can rename it to 'singleProject.ts'

**Review by RamRamez — CHANGES_REQUESTED** [review]
Great thanks @alireza-sharifpour. Just needs a tiny fix.

**Comment by alireza-sharifpour** [comment]
Thanks @RamRamez for your attention, I fixed the things you mentioned.

**Review by RamRamez — APPROVED** [review]
Thanks @alireza-sharifpour
